js/start.js

```diff
@@ -29,7 +29,7 @@ import StartupConnectMessaging from './views/StartupConnectMessaging';
 import { openSimpleMessage } from './views/modals/SimpleMessage';
 import Dialog from './views/modals/Dialog';
 import StatusBar from './views/StatusBar';
-import { getTranslationLangByCode } from './data/languages';
+import { getTranslationLangByCode, getTrumboLangFileNameByCode } from './data/languages';
```

Comment:
Everything done in this file is completely unnecessary because there's no point of integrating the trumbo translation files into polyglot because they are incompatible. Furthermore, the trumbo component is driven by the lang option when it's constructed and what is or isn't in polyglot will have no bearing on it.

I would just revert all the changes in this file. Most of the changes in the other files are relevant and should stay. I'll give more comments in those files.

Reply:
> Everything done in this file is completely unnecessary because there's no point of integrating the trumbo translation files into polyglot because they are incompatible. 

@rmisio @jjeffryes The point of what I'm doing in start.js is including the appropriate trumbowyg translation file. That is 100% necessary to do here. Should I being using a different method to programmatically include the necessary translation files?

Trumbowyg doesn't work without including the specific translation file as I'm doing here.

Reply:
@rmisio @jjeffryes Ah, I see. I didn't intend to use polyglot to include trumbowyg's translation files. BUT, I did intend to include the file - so what I'll plan to do is programmatically inject the file without using polyglot. 

Do we have a preferred method for injecting the appropriate file? Everything about what I'm doing here is still necessary, just not importing it with polyglot.